lib/fig/repository.rb

```diff
@@ -156,9 +158,9 @@ def publish_package(
 
     if not local_only
       publisher.remote_directory_for_package =
-        remote_directory_for_package(descriptor)
+        remote_directory_for_package(descriptor, true) # true = use publish URL
```

Comment:
Taken in tandem with my suggestion below this is what the call sites look like when you use keyword arguments.

```suggestion
        remote_directory_for_package(descriptor, for_publishing: true)
```

Reply:
Change integrated in 36c6405.